tools/spectral/ipa/rulesets/IPA-102.yaml

```
@@ -2,6 +2,26 @@
 # http://go/ipa/102
 
 rules:
+  xgen-IPA-102-collection-identifier-camelCase:
+    description: >-
+      Collection identifiers must be in camelCase. Logic includes:<br/>
+      - All path segments that are not path parameters<br/>
+      - Only the resource identifier part before any colon in custom method paths (e.g., 'resource' in '/resource:customMethod')<br/>
+      - Path parameters should also follow camelCase naming<br/>
+      - Certain values can be exempted via the ignoredValues configuration (e.g., 'v1', 'v2') that can be supplied as `ignoredValues` 
+      argument to the rule<br/>
+      - Paths with x-xgen-IPA-exception for this rule are excluded from validation<br/>
+      http://go/ipa/102
```

```suggestion
      Collection identifiers must be in camelCase. Logic includes:<br/>
      - All path segments that are not path parameters<br/>
      - Only the resource identifier part before any colon in custom method paths (e.g., `resource` in `/resource:customMethod`)<br/>
      - Path parameters should also follow camelCase naming<br/>
      - Certain values can be exempted via the ignoredValues configuration (e.g., 'v1', 'v2') that can be supplied as `ignoredValues` 
      argument to the rule<br/>
      - Paths with `x-xgen-IPA-exception` for this rule are excluded from validation<br/>
      http://go/ipa/102
```
[Nit] Some formatting